python/ray/_private/runtime_env/conda_utils.py

```diff
@@ -192,6 +199,8 @@ def exec_cmd_stream_to_logger(
 
     The last n_lines lines of output are also returned (stdout and stderr).
     """
+    if "env" in kwargs and _WIN32 and "PATH" not in [x.upper() for x in kwargs.keys]:
+        raise ValueError("On windows, Popen requires 'PATH' in 'env'")
```

Comment:
This change provides a nicer error message on windows when `env` is used but `PATH` is not provided. The default on windows is an empty `PATH`, which will almost always fail.